src/main/java/com/microsoft/azure/relay/HybridConnectionListener.java

```diff
@@ -381,8 +381,8 @@ public String toString() {
 		return this.cachedString;
     }
 
-	CompletableFuture<Void> sendControlCommandAndStreamAsync(ListenerCommand command, ByteBuffer buffer, Duration timeout) {
-		return this.controlConnection.sendCommandAndStreamAsync(command, buffer, timeout);
+	CompletableFuture<Void> sendControlCommandAndStreamAsync(ListenerCommand command, ByteArrayOutputStream writeBufferStream, Duration timeout) {
```

Comment:
>ByteArrayOutputStream writeBufferStream, [](start = 83, length = 40)

Does changing from ByteBuffer to ByteArrayOutputStream eliminate potential exceptions in some way? How?